.env.example

```diff
@@ -47,3 +47,8 @@ ADMIN_AUTH_TOKEN=admin
 
 # Run local integration tests:
 INTEGRATION_TARGET=http://localhost:5000/apollo
+
+# Used for to synchronize data from our DB to several webflow collections
+WEBFLOW_API_BASE_URL=https://api.webflow.com
+WEBFLOW_COLLECTION_ID=
```

Comment:
There will probably be multiple collection ids in the future, so the naming will change.